fhf-datafangst-client/src/components/MyPage/MyFollowers.tsx

```diff
@@ -0,0 +1,136 @@
+import { PersonAdd, PersonRemove } from "@mui/icons-material";
+import { Box, Grid, IconButton } from "@mui/material";
+import { VesselFilter } from "components/Filters/VesselFilter";
+import { Vessel } from "generated/openapi";
+import React, { FC } from "react";
+import {
+  useAppDispatch,
+  useAppSelector,
+  selectUser,
+  selectUserVessels,
+  setSelectedVessels,
+  updateUser,
+  selectVesselsByFiskeridirId,
+} from "store";
+import { VesselInfo } from "./VesselInfo";
+
+interface ContentProps {
+  vessels: Vessel[];
+}
+
+export const FollowList: FC = () => {
+  const dispatch = useAppDispatch();
+  const vessels = useAppSelector(selectVesselsByFiskeridirId);
+  const user = useAppSelector(selectUser);
+  const selectedVessels = useAppSelector(selectUserVessels);
```

Comment:
`selectUserVessels` does not need to be in the store since it is only used in this component.
Use `useState` instead

Reply:
At the moment it is only used in this component, but the thought was that a user could view the statistics of the vessels they follow, aswell as be able to choose some vessels which would not perist for quick statistics glances.

Reply:
Yes, but this does not contain the vessels the user follows, it contains the vessels currently selected from the dropdown.
The follow list is stored in the `User` object which is already in the store

Reply:
I know, my answer was a bit cluttered. I meant that a user might want to view some vessels statistics without actually following said vessel. For example; I might want to check a vessels statistic to see wether I find this vessel relevant. But its not super important so I can remove it.

Reply:
Lets just remove it for now.

When a page for viewing other vessels' statistics is created it can just have its own dropdown for previewing vessels with options for following/unfollowing (or this component will be moved there).